src/test/java/org/springframework/integration/kafka/config/xml/KafkaOutboundAdapterParserTests.java

```diff
@@ -60,23 +61,27 @@
 
 	@Test
 	public void testOutboundAdapterConfiguration() {
-		KafkaProducerMessageHandler<?, ?> messageHandler
+		KafkaProducerMessageHandler<?, ?> messageHandler1
 			= this.appContext.getBean("kafkaOutboundChannelAdapter.handler", KafkaProducerMessageHandler.class);
-		assertThat(messageHandler).isNotNull();
-		assertThat(messageHandler.getOrder()).isEqualTo(3);
-		assertThat(TestUtils.getPropertyValue(messageHandler, "topicExpression.literalValue")).isEqualTo("foo");
-		assertThat(TestUtils.getPropertyValue(messageHandler, "messageKeyExpression.expression")).isEqualTo("'bar'");
-		assertThat(TestUtils.getPropertyValue(messageHandler, "partitionIdExpression.expression")).isEqualTo("'2'");
-		assertThat(TestUtils.getPropertyValue(messageHandler, "sync", Boolean.class)).isTrue();
-		assertThat(TestUtils.getPropertyValue(messageHandler, "sendTimeoutExpression.expression")).isEqualTo("1000");
-
-		messageHandler
+		assertThat(messageHandler1).isNotNull();
```

Comment:
Well, I think we don't need that extra variable because we don't use the first instance after obtaining the second.
So, replacing it is a way to go